docs/changes/README.md

```diff
@@ -10,6 +10,7 @@ The `<TYPE>` corresponds to a particular schema category and should be one of:
 - `sci`: Science Pipelines schemas under continuous development (ImSim and HSC)
 - `ap`: Alert Production Database (APDB)
 - `cdb`: Consolidated Database of Image Metadata (ConsDB)
+- `efd`: ConsDB Transformed EFD
```

Comment:
I don't think we need this extra category. These changes can just he included in the cdb news fragments.